packages/next/src/server/config.ts

```diff
@@ -1,4 +1,5 @@
 import { existsSync } from 'fs'
+import { readFile } from 'node:fs/promises'
```

Comment:
Do we use the `node:` prefix anywhere else? I'm not sure this works if this specifier works in other runtimes.

Reply:
Yes. https://github.com/vercel/next.js/blob/canary/packages/next/src/server/lib/router-utils/resolve-routes.ts#L12

> I'm not sure this works if this specifier works in other runtimes.
Hmm, makes sense, hard to cover all I guess. Will remove `node:` just in case.